packages/omg-js-childchain/src/transaction/transaction.js

```diff
@@ -21,6 +21,9 @@ const MAX_OUTPUTS = 2
 const NULL_INPUT = { blknum: 0, txindex: 0, oindex: 0 }
 const NULL_OUTPUT = { owner: '0x0000000000000000000000000000000000000000', amount: 0 }
 
+const BLOCK_OFFSET = 1000000000
+const TX_OFFSET = 10000
+
```

Comment:
Im not too familiar with these two. What are the uses and are they always default to those value ?